docs/sphinx/source/user_guide/nomenclature.rst

```diff
@@ -162,8 +163,9 @@ There is a convention on consistent variable names throughout the library:
         Temperature of the module
 
     transposition_factor
-        The gain ratio of the radiation on inclined plane to global horizontal
-        irradiation: :math:`\frac{poa\_global}{ghi}`
+        The ratio of the irradiance incident on an inclined plane to the
+        the irradiance incident on a horizontal plane,
+        :math:`\frac{poa\_global}{ghi}`
```

Comment:
I would just remove `transposition_factor`.  We don't use it anywhere in pvlib AFAIK.